src/styles/_header.scss

```diff
@@ -0,0 +1,59 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-sizing: border-box;
+
+  // box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.05);
+
+  &__link {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin-bottom: 0;
+    margin-top: 0;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    text-decoration: none;
+    font-weight: 500;
+    text-transform: uppercase;
+    color: $accent-color;
+    font-size: 12px;
+    position: relative;
+    display: inline-block;
+    height: 60px;
+    line-height: 60px;
+
+    &:hover {
+      color: $is-active;
+    }
+
+    &.is-active {
+      color: $is-active;
+    }
+  }
+}
+
+.is-active::after {
```

Comment:
It seems like the `.is-active` class is being used for both text color and the after pseudo-element. Make sure that the `.is-active` class is used consistently across elements. If necessary, create separate classes for the text color and the after pseudo-element to avoid confusion and potential styling conflicts.